applications/portal/backend/api/repositories/filters_repository.py

```diff
@@ -0,0 +1,94 @@
+
+from django.db.models import Q
+from openapi.models import Sortorder
+from fastapi import HTTPException
+from portal.constants import FILTERABLE_FIELDS
+from openapi.models import Operation
+from api.services.user_service import get_current_user_id
+
+def check_filters_are_valid(filters):
+	for filter_type, filter_values in dict(filters).items():
+		if filter_type not in ["contains", "equals", "startsWith", "endsWith", "isEmpty", "isNotEmpty", "isAnyOf", "page", "search", "size", "sortOn", "operation", "isUserScope"]:
```

Comment:
Lookup in lists is not efficient, better use a set. Also, the set should be a constant, not created inline every time the function is called